Bottom: d45aa7b9758308c63248f8dbfbf581ec1db5ac37
Top:    07dba09829c088e6da86aa3f8d4c90e07ea17998
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-05 21:05:50 +0100

hid/gtk: Enable transparent lines in GL renderer


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index d9ee158..7916b41 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -79,7 +79,7 @@ ghid_set_layer (const char *name, int group, int empty)
 
   if (idx >= 0 && idx < max_copper_layer + 2)
     {
-      priv->trans_lines = false;
+      priv->trans_lines = true;
       return PCB->Data->Layer[idx].On;
     }
   if (idx < 0)
